esmvalcore/typing.py

```diff
@@ -9,7 +9,7 @@
 import numpy as np
 from iris.cube import Cube
 
-FacetValue = str | Sequence[str] | Number
+FacetValue = str | Sequence[str] | Number | bool
```

Comment:
`isinstance(True, Number)` gives me `True`, so this addition seems unnecessary, or am I missing something?